teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Set-CsTeamsSipDevicesConfiguration
+author: anmandav
+ms.author: anmandav
+online version: https://learn.microsoft.com/powershell/module/teams/set-csteamssipdevicesconfiguration
+schema: 2.0.0
+---
+
+# Set-CsTeamsSipDevicesConfiguration
+
+## SYNOPSIS
+
+This cmdlet is used to manage the organization-wide Teams SIP devices configuration.
+
+## SYNTAX
+
+```powershell
+Set-CsTeamsSipDevicesConfiguration
+   [-BulkSignIn <String>]
+   [-WhatIf]
+   [-Confirm]
+   [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to teams using Teams Sip Gateway.
+
+You must be a Teams Service Administrator or a Global Administrator for your organization to run the cmdlet.
+
+## EXAMPLES
+
+### Example 1
+In this example, Bulk SignIn is set to Disabled across the organization.
```

Comment:
Please move the description under the code block (example https://learn.microsoft.com/en-us/powershell/module/teams/new-teamchannel?view=teams-ps) and here it should say "Enabled"

Reply:
Done, moved and corrected it to Enabled